tests/docker/globus.ubuntu24.Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:24.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true
+
+#### install basic packages ####
+RUN apt-get update && apt-get install -y curl \
+    g++ \
+    gcc \
+    ftp \
+    telnet \
+    libcurl4-openssl-dev \
+    unzip \
+    python3 \
+    python3-distro \
+    python3-psutil \
+    python3-jsonschema \
+    python3-requests \
+    python3-pip \
+    python3-pyodbc \
+    wget \
+    gnupg2 \
+    lsb-release \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+
+#### Get and install iRODS repo ####
+RUN apt-get update && apt-get install -y wget gnupg2 lsb-release && \
+    wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
+    echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list && \
+    apt-get update
+
+#### Install icommands - used to set up, validate and tear down tests. ####
+#### Install externals and dev package to build the connector.         ####
+
+# Comment out the irods-icommands and irods-dev if testing against locally
+# built packages.
+RUN apt-get install -y \
+    irods-icommands \
+    irods-dev \
+    irods-externals-cmake3.21.4-\
```

Comment:
The end of L42 looks like it's missing something?

Reply:
I fixed it.

Note:  I originally tested Ubuntu 24 with the 4.3.3.  For the changes to just the Dockerfiles, I've been testing against released versions of iRODS as that is sufficient to test out these changes.  However, Ubuntu 24 was commented out since it isn't released so this was not caught.

Reply:
Sounds good.